docusaurus/docs-internals/editor-controls/data-controls/multiselect.md

```diff
@@ -0,0 +1,278 @@
+| `config?.search`    | `boolean`                                                                                                                                                                                  |    -    | If provided and set to `true`, an input field is displayed at the top to search items within the "multiSelect" control.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
```

Comment:
about `config?.search` description

i think we can remove here `"multiSelect"` because is understading by context